app/components/forms/wizard/attendee-step.js

```diff
@@ -23,15 +24,15 @@ export default Component.extend(FormMixin, {
         this.sendAction('save', this.data);
       });
     },
-    move(direction) {
+    savePublished() {
       this.onValid(() => {
-        this.sendAction('move', direction, this.data);
+        this.set('data.event.state', 'published');
```

Comment:
Here savePublished sets the event to be published, so why not just use publish method